core/whistle_services-1.0.0/src/bookkeepers/wh_bookkeeper_braintree.erl

```diff
@@ -113,16 +116,21 @@ sync([ServiceItem|ServiceItems], AccountId, Updates) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec transactions(ne_binary(), ne_binary(), ne_binary()) ->
-                          'not_found' |
-                          'unknown_error' |
-                          wh_json:objects().
-transactions(AccountId, Min, Max) ->
-    try braintree_transaction:find_by_customer(AccountId, Min, Max) of
-        Transactions ->
-            [braintree_transaction:record_to_json(Tr) || Tr <- Transactions]
+                          {'error', 'not_found'} |
+                          {'error', 'unknown_error'} |
+                          {'ok', wh_transaction:transactions()}.
+transactions(Account, Min, Max) ->
```

Comment:
Curious why you didn't follow the return pattern of `subscriptions/2` here or update `subscriptions/2` to return the 2-tuples instead of raw atom/json objects.